### ISSUE TITLE
Birch-Murnaghan equation of state

### ISSUE THREAD

**Issue by Ruoyan0103** [jochym/Elastic#71]
Thank you so much for sharing your code. I met some issues in the code. I think the equation used in file 'elastic.py' function 'BMEOS' is not Birch-Murnaghan equation. 
It is Murnaghan equation: https://en.wikipedia.org/wiki/Murnaghan_equation_of_state
The correct equation is: https://en.wikipedia.org/wiki/Birch%E2%80%93Murnaghan_equation_of_state

**Comment by jochym** [comment]
You are right. I will make the correction in the text. Later, I will implement B-M version as well. (I will also accept PR if you have time ;).
